deleteDiscordMessages.user.js

```diff
@@ -6,6 +6,12 @@
 // @match         https://discord.com/app
 // @match         https://discord.com/channels/*
 // @match         https://discord.com/login
+// @match         https://ptb.discord.com/app
```

Comment:
```suggestion
// @match         https://*.discord.com/app
```

Maybe better to cover `ptb` and `canary` together like this?
Not sure if that would include no subdomain as well.

Reply:
> Matches any URL that uses the https scheme, is on a google.com host (such as www.google.com, docs.google.com, or google.com), as long as the path starts with /foo and ends with bar
> 
> — https://developer.chrome.com/docs/extensions/mv3/match_patterns/

Looks like `*.discord.com` would actually match all 3, the base domain, and both subdomains.

Might be worth replacing the 9 entries with just 3 using `*.discord.com`. (Haven't tested it manually.)